leapp/cli/upgrade/__init__.py

```diff
@@ -14,6 +17,40 @@
 from leapp.utils.output import report_errors, beautify_actor_exception
 
 
+def logs_handler():
```

Comment:
I'd rename it to logfile_handler as this function does not handle all the logging (inlcuding logging to audit db, stdout/stderr, etc.), but it handles log files only.

Reply:
Well, as I'm thinking about it, now it does nothing else then archiving logs. So would name it `archive_logfiles()`.

Reply:
It makes sense!